db/schema.sql

```diff
@@ -2233,13 +2233,6 @@ END
 $$;
 
 
---
--- Name: FUNCTION delete_obsolete_urn_snapshot(urn_id integer, header_id integer); Type: COMMENT; Schema: maker; Owner: -
---
-
-COMMENT ON FUNCTION maker.delete_obsolete_urn_snapshot(urn_id integer, header_id integer) IS '@omit';
```

Comment:
What's up with this schema change? Could you give me the quick summary of how we use Postgraphile magic comments?

Reply:
I think we're primarily using comments to omit certain functions and tables from the postgraphile endpoint. I think that when I changed the function signature of `delete_obsolete_urn_snapshot` this comment should have been removed and `COMMENT ON FUNCTION  maker.delete_obsolete_urn_snapshot(urn_id integer, header_id integer, diff_id bigint) IS @omit` should have been added. 

The thing I'm a bit confused about is why/how ` maker.delete_obsolete_urn_snapshot(urn_id integer, header_id integer, diff_id bigint)` isn't showing up in the api, since there isn't a comment on that function in the schema at the moment. @rmulhol do you have any hunches?

Reply:
Best guess is that something we have in place to disable mutations is preventing this from bring exposed in the API but not sure 🤔

Reply:
Also I think this schema change just came from running `make test` - possible that a migration change wasn't reflected in the schema at the time the migration PR was merged. I think it should go away if you rebase at this point

Reply:
Yeah, I think you're right that this schema change is probably already in the code base, and this diff will go away with a rebase. 👍

Reply:
It looks like this is outdated but I think I can fill in - delete_obsolete_urn_snapshot is presumably VOLATILE, and because we disable the mutation plugin in postgraphile it's not going to show up. The '@omit' shouldn't be necessary.

I'd generally prefer we use the JSON in postgraphile for omitting functions instead of the SQL comments, which I think is the same view as the rest of the team.